tee-worker/cli/ts_test_vc.sh

```diff
@@ -0,0 +1,7 @@
+#!/bin/bash
+
+set -o pipefail
+
+cd /ts-tests
+yarn install
+yarn run test-vc:staging
```

Comment:
Why not merge it into `ts_test.sh`? Do we want to parallelize it? If so may be can still use the same `ts_test.sh` but different args.

Btw `ts_test_vc.sh` won't run because we miss the matching yml entry

Reply:
I was a little unfamiliar with CI configuration, and I thought that adding a new file would allow both tasks to work together, which would reduce the wait time. I'll fix it right away.